source/build.py

```diff
@@ -0,0 +1,169 @@
+import os
+import subprocess
+from os.path import join, getsize
+from HTMLParser import HTMLParser
+
+import pdb, logging
+
+
+# ============================================================================
+
+
+logging.basicConfig(level=logging.DEBUG,
+                    filename='logs.log',
+                    filemode='w')
+
+cwd = str(os.getcwd())
+lsResults = os.listdir(cwd)
+
+isMkdocsInstalled = (
+    subprocess.check_output(['mkdocs', '--version'])[0:15] == 'mkdocs, version')
+
+googleTagManagerContainerId = 'GTM-MNRH4J'
+
+googleTagManagerScriptLines = """
+<!-- Google Tag Manager -->
+  <noscript>
+    <iframe src="//www.googletagmanager.com/ns.html?id={containerId}"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe>
+  </noscript>
+  <script>
+    (function(w,d,s,l,i){{w[l]=w[l]||[];w[l].push({{'gtm.start':
+    new Date().getTime(),event:'gtm.js'}});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    }})(window,document,'script','dataLayer','{containerId}');
+  </script>
+<!-- End Google Tag Manager -->
+""".format(containerId = googleTagManagerContainerId).splitlines()
+
+
+# ============================================================================
+
+
+class PostMkdocsParser(HTMLParser):
+
+    def __init__(self, htmlFileFullPath):
+        self.reset()
+        self.results          = set([])
+        self.targetIsFullHTML = False
+        self.targetFullPath   = htmlFileFullPath
+        self.pruneThisLang    = set([])
```

Comment:
I don't know about python culture, but include word "this" feels weird to me. Even if you use "this", because it's a list and could be multiple, `pruneTheseLangs` would be better.
I would use `pruneLangs` or `pruneLangList` instead.